packages/switch/src/index.tsx

```diff
@@ -17,15 +17,6 @@ export class Switch extends MaterialComponent<ISwitchProps, ISwitchState> {
     super.componentDidMount();
     if (this.control) {
       this.MDComponent = new MDCSwitch(this.control);
-      if (this.props.value) {
-        this.MDComponent.value = this.props.value;
-      }
-    }
-  }
-
-  public componentWillUpdate(nextProps: ISwitchProps) {
-    if (nextProps.value && this.props.value !== nextProps.value) {
-      this.MDComponent.value = nextProps.value;
```

Comment:
Why was this removed?

Reply:
`value` setter is removed: https://github.com/material-components/material-components-web/tree/master/packages/mdc-switch#mdcswitch-properties-and-methods

Reply:
We should maybe think of an alternative

Reply:
shouldn't `checked` be fine?

Reply:
yes it should